main.py

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/python3
+import sys,json
+sys.path.append('./modules')
+sys.path.append('./modules/MySqlDB')
+import AptitudeEngine
+import question_analyser as q
+import aptitude_engine_db as ae
+from flask import Flask, render_template,url_for,request
+
+app = Flask(__name__)
+analyser = False
+dataset_labels = ["number_series","relative_speed","clock","lcm","TimeWork"]
+dataset_location ="/home/ashish/AptitudeEngine/AptitudeEngine/modules/Dataset/jsons"
+dataset_files = ["numseries.json","train.json","clock.json","lcm.json","TimeWork.json"]
+
+@app.after_request
+def add_header(response):
+    """
+    Add headers to both force latest IE rendering engine or Chrome Frame,
+    and also to cache the rendered page for 10 minutes.
+    """
+    response.headers['X-UA-Compatible'] = 'IE=Edge,chrome=1'
+    response.headers['Cache-Control'] = 'public, max-age=0'
+    return response
+@app.route('/')
```

Comment:
Add two newline above 
```python
@app.route
```
Everywhere, wherever you have not added newlines after end of function